src/Phoenix/PhoenixClient.ts

```diff
@@ -25,6 +25,10 @@ export class PhoenixClient {
   public async containerConnectionInfo(
     provisionData: IProvosionData
   ): Promise<IPhoenixResponse<IPhoenixConnectionInfoResult>> {
+    const connectionStatus: DataModels.ContainerConnectionInfo = {
+      status: ConnectionStatusType.Allocating,
+    };
+    useNotebook.getState().setConnectionInfo(connectionStatus);
     const response = await window.fetch(`${this.getPhoenixContainerPoolingEndPoint()}/provision`, {
```

Comment:
surround with try catch block, set status to Failed if it hits an exception

Reply:
added